homeassistant/components/openai_conversation/const.py

```diff
@@ -60,6 +60,15 @@
     "o3-mini",
 ]
 
+UNSUPPORTED_IMAGE_MODELS: list[str] = [
```

Comment:
I wonder if we should turn it around. Only list supported ? Given that they launched 5 and it didn't support it, we now would flag in 6 as supporting it when it drops, while it might not.